.travis.yml

```diff
@@ -30,15 +30,12 @@ script:
   - make container
   - kubectl cluster-info
   - |
-    vR=`echo $TRAVIS_BRANCH |awk -F 'velero_'  '{print $2}'`
-    if [ -z $TRAVIS_BRANCH ] || [ $TRAVIS_BRANCH == "master" ] || [ -z $vR ]; then
-      export VELERO_RELEASE=v1.0.0
-      # In travis we use openebs operator from openebs/openebs:master only.
-      export OPENEBS_RELEASE="master"
+    export VELERO_RELEASE=v1.0.0
+    if [ ! -z $TRAVIS_TAG ]; then
+      export OPENEBS_RELEASE=${TRAVIS_TAG}
```

Comment:
If the travis tag is 1.9.0-RC1, which openebs operator will it download?

Is this used for downloading the YAML file from openebs/openebs. The openebs/openeb repo will not be tagged for RC builds and also it will be tagged with v1.9 at the end of the release.

Reply:
yes, so we are using yaml file from master branch only. Last time we had a discussion about this issue, we thought of improving yaml file to some other format where we can pass the variable to decide image tag for other component.